src/reshape.js

```diff
@@ -31,3 +32,19 @@ export function reshape(input, newShape) {
   const output = new Tensor(outputShape, input.data);
   return output;
 }
+
+/**
+ * Reduce the rank of a tensor by eliminating dimensions with size 1 of the tensor shape.
+ * @param {Tensor} input
+ * @param {MLSqueezeOptions} options
+ * @return {Tensor}
+ */
+export function squeeze(input, {axes} = {}) {
+  validateSqueezeParams(...arguments);
+  const inpAxes = axes ?? new Array(input.rank).fill(0).map((_, i) => i);
```

Comment:
```suggestion
  const inputAxes = axes ?? new Array.from({length: input.rank}, (_, i) => i);
```

Hmm, kinda confusing way of initializing a sequence by chaining multiple methods together (took me a bit to figure out), rather than just saying `Array.from({length: input.rank}, (_, i) => i)`, which is longer character-wise, but it's clearer intention-wise (and probably more performant than a fill which is overwritten by a map). This one surprising case where C++ is shorter :b `std::ranges::iota(inputAxes, 0)`.

Reply:
I followed the previous implementation in squeeze.js. And there occured a typeERROR  when I changed the code:Array.from is not a constructor.

Reply:
I followed the previous implementation in squeeze.js. And there occured a typeERROR when I changed the code:Array.from is not a constructor.